R/expr-expr.R

```diff
@@ -6,12 +6,11 @@
 #' An expression is a tree of operations that describe how to construct one or more [Series].
 #' As the outputs are [Series], it is straightforward to apply a sequence of expressions each of
 #' which transforms the output from the previous step.
-#' See examples for details.
-#' @name polars_expr
-#' @aliases Expr expression
```

Comment:
Please do not remove the aliases.
Same as https://github.com/eitsupi/neo-r-polars/pull/18#discussion_r1804634224